docs/solidity-by-example.rst

```diff
@@ -36,7 +36,7 @@ of votes.
 
 ::
 
-    pragma solidity ^0.4.16;
+    pragma solidity ^0.4.21;
```

Comment:
Is this change needed?

Reply:
In the issue they mention that this example was still with ^0.4.16 while the other ones were updated to ^0.4.21. 
If you prefer i can leave it as ^0.4.16.

Reply:
If it works with 0.4.16, it should stay at 0.4.16.